tests/test_security_oauth2_scopes.py

```diff
@@ -0,0 +1,51 @@
+from fastapi import FastAPI, Security
+from fastapi.params import Depends
+from fastapi.security import OAuth2PasswordBearer
+from fastapi.security.oauth2 import SecurityScopes
+from fastapi.testclient import TestClient
+
+app = FastAPI()
+
+oauth2_scheme = OAuth2PasswordBearer(
+    tokenUrl="token",
+    scopes={"me": "Read information about the current user.", "items": "Read items."},
```

Comment:
Scope definition syntax: https://datatracker.ietf.org/doc/html/rfc6749#section-3.3
(I express doubts regarding the correctness of the `scopes`)